Description: minimal setup module
 Keep setup.pm a simple as possible so it is simply updated with expected paths
 during the build stage.
Author: Guillaume Bougard <gbougard@teclib.com>
Forwarded: not-needed
Last-Update: 2018-10-08

--- a/lib/setup.pm
+++ b/lib/setup.pm
@@ -5,9 +5,6 @@ use strict;
 use warnings;
 use parent qw(Exporter);
 
-use File::Spec;
-use Cwd qw(abs_path);
-
 our @EXPORT = ('%setup');
 
 our %setup = (
@@ -16,22 +13,4 @@ our %setup = (
     vardir  => './var',
 );
 
-# Compute directly libdir from this setup file as it should be installed
-# in expected directory
-eval {
-    $setup{libdir} = abs_path(File::Spec->rel2abs('..', __FILE__))
-        unless ($setup{libdir} && File::Spec->file_name_is_absolute($setup{libdir}));
-
-    # If run from sources, we can try to rebase setup keys to absolute folders related to libdir
-    if (File::Spec->file_name_is_absolute($setup{libdir})) {
-        foreach my $key (qw(datadir vardir)) {
-            # Anyway don't update if target still absolute
-            next if ($setup{$key} && File::Spec->file_name_is_absolute($setup{$key}));
-
-            my $folder = abs_path(File::Spec->rel2abs('../'.$setup{$key}, $setup{libdir}));
-            $setup{$key} = $folder if $folder && -d $folder;
-        }
-    }
-};
-
 1;
